MMA Removal Utility/SetupCleanUp.md

```diff
@@ -0,0 +1,59 @@
+# **Steps to clean up AzTS MMA Removal Utility**
+
+In this section, we will walk you through the steps of cleaning up or deleting resources associated with AzTS MMA Removal Utility.
+
+Before initiating the deletion please validate the prerequisites, download and extract deployment package using [Prerequisites](./Prerequisites.md).
```

Comment:
Before initiating the deletion please validate the **prerequisites** - here we can provide link to prereq
download and extract deployment package using **[Prerequisites]** - here we can provide link to download and extract the deployment package.

Reply:
Asim, actually both the content is withing the same file, so pointing to that single file.

Reply:
Sub section comments could be used. But its just minor suggestion. You could skip